jdaviz/app.py

```diff
@@ -3078,6 +3090,61 @@ def _create_tray_item(self, tray_registry_member):
         }
         return tray_item
 
+    def update_new_viewers_from_registry(self):
+        # TODO: implement jdaviz.new_viewers dictionary to instantiated items here
+        if self.config != 'deconfigged':
+            raise NotImplementedError("update_new_viewers_from_registry is only "
+                                      "implemented for the deconfigged app")
```

Comment:
Suggestion: let's avoid making "deconfigged" part of the user vocabulary when it's not necessary.
```suggestion
                                      "implemented for {self._jdaviz_helper.__class__}.")
```

Reply:
Agreed - but there is no way to get here from public API, so no one should see this unless they go digging for it